source/SavedGame.cpp

```diff
@@ -48,6 +50,8 @@ void SavedGame::Load(const string &path)
 			planet = node.Token(1);
 		else if(node.Token(0) == "playtime" && node.Size() >= 2)
 			playTime = Format::PlayTime(node.Value(1));
+		else if(node.Token(0) == "flagship index" && node.Size() >= 2)
+			flagshipTarget = node.Value(1);
```

Comment:
Is this the first "dependent" node? As in, it needs to appear before any `"ship"` node or else it doesn't work?

Reply:
It'll need to appear up top, yeah. However, if a save file does not have that node (like with an old save file), it'll default to the original behavior.